components/elements/ElementPresentation.vue

```diff
@@ -18,20 +18,24 @@
       <Cta
         v-if="to"
         class="element-presentation__get-started-cta"
-        :to="to"
-        @click="segmentAction && $trackClickEvent({
-          action: segmentAction
-        })"
+        :link="{
```

Comment:
When removing `link`, use the property form for the cases when you are writing the name of the properties explicitly, like this one:
```suggestion
        :url="to"
        :segment="{ action: segmentAction }"
```

Reply:
I prefer using `v-bind`